lib/trie/substrate/SubstrateTrieDB.sol

```diff
@@ -99,7 +101,7 @@ contract SubstrateTrieDB is TrieDB {
         }
 
         for (uint256 i = 0; i < 16; i ++) {
-            NodeHandleOption handle;
+            // NodeHandleOption handle;     // shadows existing `handle` variable
```

Comment:
Let's rename the variable, we shouldn't re use the old one

Reply:
what name would you suggest? im thinking `childHandle`

Reply:
Yeah that works